cupy/cuda/common.pxd

```diff
@@ -11,3 +13,6 @@ cdef extern from '../cuda/cupy_common.h':  # thru parent to import in core
     ctypedef unsigned long long cpy_ulong
     ctypedef float cpy_float
     ctypedef double cpy_double
+    ctypedef struct cpy_complex64 'cuComplex'
+    ctypedef struct cpy_complex128 'cuDoubleComplex'
+    ctypedef bool_t cpy_bool 'bool'
```

Comment:
Adding `cdef struct` and `ctypedef` as following would be better?

```Python
cdef extern from '../cuda/cupy_common.h':  # thru parent to import in core
    cdef struct cuComplex
    cdef struct cuDoubleComplex
    ...
    ctypedef cuComplex cpy_complex64
    ctypedef cuDoubleComplex cpy_complex128
    ctypedef bool_t cpy_bool
```

Reply:
Thanks, @takagi! Question: Aren't cuComplex and cuDoubleComplex imported from cupy_common.h? I think based on Cython's doc this usage is legit (see the "or" part in the 3rd row):
https://cython.readthedocs.io/en/latest/src/userguide/external_C_code.html#styles-of-struct-union-and-enum-declaration

Reply:
I thought C preprocesser would finally expand `#include cupy_cuComplex.h` to the declarations of `cuComplex` and `cuDoubleComplex`, so I could import those here. But, yes, now I understand the current code is valid. You don't need to make a change.